UnityProject/Assets/Scripts/ChemistryComponents/ReagentContainer.cs

```diff
@@ -438,13 +438,24 @@ public bool WillInteract(ContextMenuApply interaction, NetworkSide side)
 
 		public void ServerPerformInteraction(ContextMenuApply interaction)
 		{
+			var eyeItem = interaction.Performer.GetComponent<Equipment>().GetClothingItem(NamedSlot.eyes).GameObjectReference;
 			switch (interaction.RequestedOption)
 			{
 				case "Contents":
-					// I think some condition should be met before the user knows what the exact contents of a container are.
-					// Wearing science goggles?
-					ExamineContents();
-					break;
+					{
+						if (eyeItem)
```

Comment:
Perhaps it would be a good idea to not show the context menu option if the goggles aren't worn (perhaps ignore admin, and skip checks here to allow admin to use it? Might be useful) see this class's `GenerateRightClickOptions()`.

Reply:
If there are no goggles you can still _examine_ but it will only give you basic info like "Half full with green powder" or whatever the container is permitted to expose to the player.

Reply:
Hmm ok. Would've thought this class'd be implementing `IExaminable` actually. Alright.